frontend/lib/src/components/widgets/Form/FormSubmitButton.test.tsx

```diff
@@ -15,6 +15,7 @@
  */
 
 import { screen } from "@testing-library/dom"
+import "@testing-library/jest-dom"
```

Comment:
extreme nit: probably best to put the side effect import at the very top (and potentially with a newline separating it from other imports)